stories.md

```diff
@@ -1,103 +1,103 @@
+Stories help us to organize our work. They need to comply to certain requirements to be effective though.
   
-![](http://2.bp.blogspot.com/_H0iqHTCqRyo/R4zyNrT94KI/AAAAAAAAATo/VY8M-kc1MyU/s400/user+stories+-+agile+software+development+-+cards.jpg)
-
-A story normally represents the view of a person, e.g. I as product owner for product X would like to see Y features to be implemented because ... e.g. customer Z needs this, or strategic, ...
+A story normally represents the view of a person, e.g. "I as product owner for product X would like to see Y features to be implemented because... customer Z needs this... our strategy is..."
 
 
 ### Properties of a good story
 
 - Well chosen name
 - Clear description of what we want to achieve
-- Priority, is a label: Critical, Urgent, Normal or Minor
-- Milestone (only 1, date when this story has to be delivered)
+- Priority set a label:
+ - Critical
+ - Urgent
+ - Normal
+ - Minor
+- Milestone: is a label, linking the story to a specific release date, shared with other stories
 - Assignee: is the [story card owner](roles.md)
-- Effort (in days)
-   - Put as ```(effort:4)``` anywhere in body of story 
-- Clear set of requirements or link to another document which has the requirements (in same repo)
-- Clear set of acceptance tests or link to another document which has these acceptance tests (in same repo)
+- Estimnation of the effort in days or hours
+- Estimated Time of Arrival (ETA) is **daily** reviewed estimation of a spefic date when work will be finished
+- Clear set of requirements or link to another document in the same repository listing and describing all requirements
+- Clear set of acceptance tests or link to another document in the same repository which has the acceptance tests detailed
 - Links (URLs) in same repo to
-  - Spec
+  - Specifications
   - Product Requirements Document (PRF) if relevant
-  - Features or bugs which are related to it (milestone as used on specific repo)
+  - Features or bugs which are related to the story
 - Format:
 	- Title: ```Story title``` (```Story name```)
-	- The story name is used to referenced in [Tasks](./tickets_tasks.md) related to the story
+	- The story name is used as reference in [Tasks](./tickets_tasks.md) related to the story
 
 
-### stories can constantly be re-prioritized
+### Stories can constantly be re-prioritized
 
 ![](http://agilemodeling.com/images/requirementsManagement.gif)
 
 
-### Story format:
+### Story format
 
 #### Title: ```$storyDescription ($storyname)```
 
-E.g.: ```fix user escalated UI issues (ui_issues)```
-
-	
-### Body:
-
+#### Body
 
 ```
 ## GOAL:
 
-- Clearly describe the goal what needs to be achieved when the story is delivered.
+Clearly describe what needs to be achieved.
 
 ## DESCRIPTION
 
-- describe what story is about
+Describe what story is about.
 
-## REQUIREMENTS
+## LINKS:
 
-- Add specifications, remarks, implementation hints, requirements
-- IMPORTANT
+- Tasks: link to a Kanban board, using waffle.io
```

Comment:
link to kanban of story:
https://waffle.io/gig-projects/org_development?search=js8beta:&label=type_task